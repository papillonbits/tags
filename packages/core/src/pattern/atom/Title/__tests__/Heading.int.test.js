import { h1, h2, h3, h4 } from './Heading.int.story'

describe('<Heading />', () => {
  describe('Render', () => {
    test('must match h1', () => {
      expect(global.renderToJSON(h1())).toMatchSnapshot()
    })

    test('must match h2', () => {
      expect(global.renderToJSON(h2())).toMatchSnapshot()
    })

    test('must match h3', () => {
      expect(global.renderToJSON(h3())).toMatchSnapshot()
    })

    test('must match h4', () => {
      expect(global.renderToJSON(h4())).toMatchSnapshot()
    })
  })
})
