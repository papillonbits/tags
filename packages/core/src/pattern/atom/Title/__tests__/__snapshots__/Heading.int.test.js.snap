// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> Render must match h1 1`] = `
<div>
  <span
    class="heading headingH1"
  >
    some heading
  </span>
</div>
`;

exports[`<Heading /> Render must match h2 1`] = `
<div>
  <span
    class="heading headingH2"
  >
    some heading
  </span>
</div>
`;

exports[`<Heading /> Render must match h3 1`] = `
<div>
  <span
    class="heading headingH3"
  >
    some heading
  </span>
</div>
`;

exports[`<Heading /> Render must match h4 1`] = `
<div>
  <span
    class="heading headingH4"
  >
    some heading
  </span>
</div>
`;
